### ISSUE TITLE
EarTrumpet: "The data area passed to a system call is too small"

### ISSUE THREAD

**Issue by btastic** [File-New-Project/EarTrumpet#306]
Getting the following error since today:

![image](https://user-images.githubusercontent.com/6894999/48666705-24b47d80-eac7-11e8-808f-30cc96775304.png)

Windows 10, version 1809

I tried repairing, resetting, uninstalling, reinstalling.. nothing did work

**Comment by riverar** [comment]
Thanks @btastic passing this immediately to Microsoft.

**Comment by btastic** [comment]
Hey there, I just did that. Hope it helps!

**Comment by riverar** [comment]
@btastic Thanks, can you share a link here?

**Comment by btastic** [comment]
@riverar I am not sure what you need? I just sent it yesterday and I didn't pay attention for a link

**Comment by riverar** [comment]
@btastic The Feedback Hub sucks in that it doesn't immediately index content. So without a link to your item, I can't past it on to Microsoft. You can find your feedback under My Feedback in the Feedback Hub then click the Share button in the top right corner to get a link.

Thanks!
![image](https://user-images.githubusercontent.com/475132/48792810-14102d00-ecaa-11e8-85a0-f08b3e338018.png)

**Comment by btastic** [comment]
Alright, had to resubmit because I wasn't logged in. https://aka.ms/AA399kl

**Comment by riverar** [comment]
@btastic Ah, that's a pain. Thanks for doing that again, passing it on now.

**Comment by riverar** [comment]
![image](https://user-images.githubusercontent.com/475132/48793468-e3c98e00-ecab-11e8-9676-e3ed33c63b9f.png)

**Comment by btastic** [comment]
Yea sure..

![image](https://user-images.githubusercontent.com/6894999/48793547-95da7780-ecf7-11e8-935b-4e9824812847.png)

**Comment by riverar** [comment]
Got it, thanks. I see what's going on now (different region). We can't directly see that but I can send the link to Microsoft. Stay tuned!

**Comment by btastic** [comment]
Alright no worries, just happy to help to get EarTrumpet working again :)

**Comment by riverar** [comment]
@btastic Sorry to keep bothering you, Microsoft is asking if can you file another issue under:
“Developer Platform > App Deployment” or “Install and Update > App launch issues”

This will gather the right logs needed.

**Comment by btastic** [comment]
There you go: https://aka.ms/AA395pj

**Comment by riverar** [comment]
@btastic Microsoft has narrowed it down but needs a little more info from your machine. Can you email us (team@eartrumpet.app) and I'll send you a script to run offline? Thanks very much for the help.

**Comment by btastic** [comment]
Sure just sent you an email

**Comment by btastic** [comment]
I'd like to keep some of the communication here in case anyone else stumbles upon this issue: (I simplified the steps)

1. @riverar sent me an email containing some proprietary powershell scripts for logging
2. I ran the script, started EarTrumpet - crashed
3. I ran the script again to stop it
4. Uploaded the generated file for further investigation

**Comment by riverar** [comment]
Sent follow up email to Microsoft. Awaiting response.

**Comment by btastic** [comment]
Hey there, just had the issue that I wanted to use the latest PowerBi version, which forced me to get it via windows store (note: I never use the windows store for apps)

I was surprised that I got the same error message. So I decided to dig around the internet for a bit, and it seems to be an issue with the SQL Server I have installed locally, which also has the filestream feature enabled.

After disabling the filestream feature, everything is back to normal. I can launch PowerBi from the store, and most importantly, I can finally use EarTrumpet again!

I found it via: http://ilikesqldata.com/centennial-apps-desktop-bridge-sql-server-and-error-the-data-area-passed-to-a-system-call-is-too-small/

I still need the filestream feature, but I guess microsoft is working on it ™
